public/app/features/alerting/unified/rule-list/hooks/useFilteredRulesIterator.ts

```diff
@@ -111,11 +127,11 @@ function groupFilter(group: PromRuleGroupDTO, filterState: RulesFilter): boolean
   const { name, file } = group;
 
   // TODO Add fuzzy filtering or not
-  if (filterState.namespace && !file.includes(filterState.namespace)) {
+  if (filterState.namespace && !file.toLowerCase().includes(filterState.namespace)) {
```

Comment:
should we also lowercase `filterState.namespace`?

Reply:
ok, i see that is normalized before. Maybe we should be sure that this precondition is always true? not sure how though

Reply:
The filter here is rather temporary. I didn't want to invest much time to adapt the code we already have for the old list page. I expect we'll eventually remove this filter and use the old one